crates/proof-of-sql/src/base/scalar/scalar_ext.rs

```diff
@@ -76,9 +76,9 @@ pub(crate) fn test_scalar_constants<S: Scalar>() {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use crate::{
-        base::scalar::{test_scalar::TestScalar, Curve25519Scalar, MontScalar},
-        proof_primitive::dory::DoryScalar,
+    use crate::base::scalar::{
+        test_scalar::{TestScalar, TestScalar as DoryScalar},
```

Comment:
I'd prefer for DoryScalar to get replaced by TestScalar in the tests rather than done via an alias. This is confusing.

Reply:
Oh, I was supposed to replace the DoryScalar with TestScalar. I made it such to check if tests pass with the changes. I will fix this one and other ones